src/LocalTime.php

```diff
@@ -662,16 +663,12 @@ public function jsonSerialize(): string
      */
     public function __toString(): string
     {
-        if ($this->nano === 0) {
-            if ($this->second === 0) {
-                return sprintf('%02u:%02u', $this->hour, $this->minute);
-            } else {
-                return sprintf('%02u:%02u:%02u', $this->hour, $this->minute, $this->second);
-            }
-        }
-
-        $nanos = rtrim(sprintf('%09u', $this->nano), '0');
+        $hasNanos = $this->nano !== 0;
```

Comment:
Here we could go with `> 0` too. I'm not sure what I prefer. @BenMorel An opinion ?

Reply:
I would go with `!== 0`, and I would inline the condition. I suspect this may be (very slightly) faster, as well.